Resources/Prototypes/Entities/Objects/Fun/water_balloons.yml

```diff
@@ -0,0 +1,107 @@
+- type: entity
+  parent: BaseItem
+  id: EmptyWaterBalloon
+  suffix: Empty
```

Comment:
i think this suffix is not really needed here because "empty" already stands in the name

Reply:
fair enough, i added the sufix way back before changing the water balloon name like a bit ago lol